camera.py

```diff
@@ -0,0 +1,430 @@
+#***************************************************************************
+#*                                                                         *
+#*   Copyright (c) 2020 Howetuft <howetuft@gmail.com>                      *
+#*                                                                         *
+#*   This program is free software; you can redistribute it and/or modify  *
+#*   it under the terms of the GNU Lesser General Public License (LGPL)    *
+#*   as published by the Free Software Foundation; either version 2 of     *
+#*   the License, or (at your option) any later version.                   *
+#*   for detail see the LICENCE text file.                                 *
+#*                                                                         *
+#*   This program is distributed in the hope that it will be useful,       *
+#*   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+#*   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+#*   GNU Library General Public License for more details.                  *
+#*                                                                         *
+#*   You should have received a copy of the GNU Library General Public     *
+#*   License along with this program; if not, write to the Free Software   *
+#*   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+#*   USA                                                                   *
+#*                                                                         *
+#***************************************************************************
+
+"""This module implements a Camera object, which allows to take a snapshot
+of Coin Camera settings and use them later for rendering"""
+
+from collections import namedtuple
+from math import degrees, radians
+import shlex
+
+from PySide2.QtWidgets import QAction
+from PySide2.QtCore import QT_TRANSLATE_NOOP, QObject, SIGNAL
```

Comment:
I think it would be better for you to import from PySide, not from PySide2.

This would be to provide some support for Qt4, although of course Qt5 should be preferred. For a longer explanation see my comment here: [QTgui.QDialog not working](https://forum.freecadweb.org/viewtopic.php?f=22&p=364598#p364598)

Reply:
OK, it's done. I've squashed the modifications in my previous linting commit...